.github/actions/install-tools/action.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+name: install-tools
+description: "Install pipeline tools"
```

Comment:
For the future might be good to add a comment or modify this description to note that they are used by goreleaser so that we don't nuke them again accidentally down the line.